### ISSUE TITLE
want proxy example

### ISSUE THREAD

**Issue by sreboot** [arekinath/triton-dehydrated#34]
Small improvement to include proxy server option in the default config file.

Example:
```
# Proxy server to use for letsencrypt.org APIs
# CURL_OPTS="-x http://proxy:3128"
```

Useful for cases where direct Internet connectivity is not available.

**Comment by sreboot** [comment]
Part of PR https://github.com/joyent/triton-dehydrated/pull/32/files#diff-2245023265ae4cf87d02c8b6ba991139R18-R20

**Comment by bahamat** [comment]
Fixed in #32
